### PR TITLE
Support default test.sh parameter by CMD

### PR DESCRIPTION
**Pull request by sophia-guo**
Support default test_options as the test suites or targets per
applications in docker image.
Keep --testtarget as user defined test suites or targets which override
the default one, which is helpful for debugging or setting different
suites for different jdk version or vendors.
Update $testtarget with double quote as docker run parameters to avoid
the string is cut by space to fix the issue mentioned https://github.com/adoptium/aqa-tests/issues/1211#issuecomment-1058532135

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Grinder scala with default CMD setting https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3985/
Grinder with personalized testsuite https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3986/

File: external/external.sh [hunk 1]
```diff
@@ -180,11 +180,19 @@ if [ $command_type == "run" ]; then
 	fi
 	if [ $reportsrc != "false" ]; then
 		echo "docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type $testtarget"
-		docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type $testtarget;
+		if [ -n "$testtarget" ]; then
+			docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type "$testtarget";
+		else
+			docker run $mountV --name $test-test adoptopenjdk-$test-test:${JDK_VERSION}-$package-$docker_os-${JDK_IMPL}-$build_type;
+		fi
```
Comment (llxia) [inline external/external.sh:187]: I am not very clear why we need if-else here

Comment (sophia-guo) [reply external/external.sh:187]: As $testtarget may be the string include spaces we have to use "$testtarget" as parameter, which also means it could be an empty string. In that case the empty string may override the default setting, which is not what we expected. We could also try to test if $testtarget is null instead of an empty string, but I don't think that helps a lot.

File: external/payara-mp-tck/test.properties [hunk 2]
```diff
@@ -1,6 +1,7 @@
 github_url="https://github.com/payara/MicroProfile-TCK-Runners.git"
 test_command="mvn --batch-mode clean verify"
 test_results="testResults"
+test_options="-pl !MicroProfile-OpenTracing/tck-runner,!MicroProfile-JWT-Auth,!MicroProfile-JWT-Auth/tck-arquillian-extension,!MicroProfile-JWT-Auth/tck-runner"
```
Comment (smlambert) [inline external/payara-mp-tck/test.properties:4]: At some point it may be interesting to try reincluding these, but definitely not needed as part of this PR

**Review by smlambert — APPROVED** [review]

**Review by llxia — APPROVED** [review]
